docs/topics/cache.txt

```diff
@@ -169,6 +169,128 @@ particularly temporary.
+username and password, add them in the ``LOCATION`` along with the URL::
+
+    CACHES = {
+        'default': {
+            'BACKEND': 'django.core.cache.backends.redis.RedisCache',
+            'LOCATION': 'rediss://username:password@127.0.0.1:6379',
+        }
+    }
+
+If you have multiple redis servers set up in replication mode, you can specify
+the servers either as a semicolon or comma delimited string, or as a list.
+While using multiple servers, all the writes are performed on the lead server.
+Read operations are performed on one of the replicas chosen at random. The first
+server address must be the leader server and the other server addresses must be
+the followers (replicas of leader)::
+
+    CACHES = {
+        'default': {
+            'BACKEND': 'django.core.cache.backends.redis.RedisCache',
+            'LOCATION': [
+                "redis://127.0.0.1:6379", # leader
+                "redis://127.0.0.1:6378", # read-replica 1
+                "redis://127.0.0.1:6377", # read-replica 2
+            ],
+        }
+    }
+
+Along with the standard :ref:`cache_arguments`, the redis caching backend also
+accepts the following additional options:
+
+* ``db``: By default, redis ships with 16 logical databases. A database of your
+  choice can be selected either by setting it in the ``LOCATION`` as querystring
+  or path argument, or by setting it in the ``OPTIONS``. The default choice for
+  the database is 0. We can set the database in the following manner::
+
+    CACHES = {
+        'default': {
+            'BACKEND': 'django.core.cache.backends.redis.RedisCache',
+            'LOCATION': 'redis://127.0.0.1:6379',
+            'OPTIONS': {
+                'db': '10'
+            }
+        }
+    }
+
+  Read more about how `database selection`_.
+
+  .. _`database selection`: https://redis-py.readthedocs.io/en/stable/#redis.ConnectionPool.from_url
+
+  Also note that in a multi redis server setup, specifying different databases
+  in the URL would not work as the redis replication works only on the same
+  database number across the servers, hence the same database must be selected.
+  The ``db`` argument in ``OPTIONS`` can be used in such cases to avoid
+  confusion.
+
+* ``pool_class``: Connection pools create a set of connections which are
+  retrieved as and when required. You can set it to the connection pool of your
+  choice or even develop a custom connection pool. This defaults to
+  ``redis.ConnectionPool``. Read more about `connection-pools`_.
+
+  .. _`connection-pools`: https://github.com/andymccurdy/redis-py#connection-pools
+
+* ``parser_class``: Parser classes control on how the responses from the
+  Redis server are parsed. By default ``hiredis`` will be used if installed
+  otherwise the ``PythonParser`` will be used. ``Hiredis`` is written in C and
+  therefore can give up to 10x gains over PythonParser.
+  Read more about `parsers`_.
+
+  .. _`parsers`: https://github.com/andymccurdy/redis-py#parsers
```

Comment:
```suggestion
* ``parser_class``: Parser classes control on how the responses from the
  Redis server are parsed. By default ``hiredis`` will be used if installed
  otherwise the ``PythonParser`` will be used. ``Hiredis`` is written in C and
  therefore can give up to 10x gains over the ``PythonParser``.
  See the ``redis-py`` docs for more about `parsers`_.

  .. _`parsers`: https://github.com/andymccurdy/redis-py#parsers
```